src/System.Private.Xml/tests/Xslt/XslCompiledTransformApi/Errata4.cs

```diff
@@ -266,6 +267,7 @@ private string GenerateStringToInject(string charsToChooseFrom, int position, Ch
             }
         }
 
+        [ActiveIssue(14775)]
```

Comment:
I believe only the test that pass true as the first argument are failing, but I guess we can disable the whole test for now just to be safe.